ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -89,6 +89,21 @@
   @Default("false")
   public final boolean storeEnableHardDelete;
 
+
+  /**
+   * How many disks should be started in parallel.
+   */
+  @Config("store.num.disk.startup.threads")
+  @Default("10")
```

Comment:
I don't think there is configuration required for within disks either. Is there a valid case for not wanting the maximum parallelism possible?

Looking at it from another angle, how would someone decide what value to set for the configuration? It doesn't affect performance.